File: app/src/main/java/com/rahbarbazaar/homadit/android/models/edit_products/Bought.java
package com.rahbarbazaar.homadit.android.models.edit_products;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class Bought {
    @SerializedName("data")
    @Expose
    public List<EditProducts> data = null;
}

File: app/src/main/java/com/rahbarbazaar/homadit/android/models/shopping_product/Bought.kt
package com.rahbarbazaar.homadit.android.models.shopping_product

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName



class Bought {
    @SerializedName("data")
    @Expose
    var data: List<ShoppingProductList>? = null
}